### ISSUE TITLE
feat: Easier way to configure a different assets prefix in a game class.

### ISSUE THREAD

**Issue by erickzanardo** [flame-engine/flame#2804]
# Problem to solve
Right now, we can't change the `prefix` attribute of an `AssetsCache` instance, since it has no setter.

# Proposal

We should change that class to include a setter so it will be consistent with `Images` and even `AudioCache`.

# More information

**Comment by spydon** [comment]
I'm not sure it is the right way to add a helper method for this as deep as in the `Game` class when not many has (if anyone?) has asked for it? It feels a bit YAGNI, wouldn't it be better to confine the cache methods within the cache classes instead of proxy them through the game?
Maybe what is actually missing is good docs for how to use the current methods, since they already are very easy to use?
```
images.prefix = "new_prefix";
assets.prefix = "new_prefix"; // Should exist, but doesn't
```
Just realized that the `assets.prefix` setter should be created, since it only exists on `Images` currently.

**Comment by erickzanardo** [comment]
>  (if anyone?) has asked for it?

Well, I am 😅

> Just realized that the assets.prefix setter should be created, since it only exists on Images currently.

Yeah, that definitely is something that we should do, cause that is one of the reasons for the confusion IMO, since in images you can just set, and on assets cache you need to assign a new instance.

My reasoning with having a single method to change the prefixes is that it is easier to forget to make two calls (assets.prefix = '' and images.prefix = ''), than a single one, I feel this is specially beneficial in tests, Where we might want to create test game classes more often, while in the real sourcebase, we will do this usually once.

But I am also fine if we just change the `AssetsCache.prefix` to have a setter and document it if needed, I feel this will improve the confusion a bit.

**Comment by spydon** [comment]
> But I am also fine if we just change the AssetsCache.prefix to have a setter and document it if needed, I feel this will improve the confusion a bit.

I would definitely go with this instead of a proxy method. :)

What do you think, should we close this issue and open a new one about the `AssetsCache.prefix`, or re-formulate this one?

**Comment by erickzanardo** [comment]
I can reformulate this one, and then mark our comments as hidden

**Comment by erickzanardo** [comment]
Fixed by #2806
